src/plone/app/theming/browser/controlpanel.py

```diff
@@ -188,6 +200,8 @@ def update(self):
 
             for line in parameterExpressionsList:
                 try:
+                    if isinstance(line, six.binary_type):
```

Comment:
I have tried this on standard Plone 5.2 on Py 2 and 3. The change works. But I don't completely trust it for Python 2. If the line is binary, you decode it to unicode, but afterwards the code calls `str` on it, making it binary again. In fact when I try it in Python 2, the line is already unicode, so this change has no effect. But I still don't trust it. Please do this on Python 3 only:

```suggestion
                    if six.PY3 and isinstance(line, six.binary_type):
```